navigator/backend/src/main/scala/com/digitalasset/navigator/dotnot/package.scala

```diff
@@ -44,10 +44,6 @@ package object dotnot {
   final case class UnknownType(name: String, cursor: PropertyCursor, value: String)
       extends DotNotFailure
 
-  type NameMatcher = String => Boolean
-  type ValueMatcher = String => Boolean
-  type Action[T, R, C] = (T, PropertyCursor, String, C) => Either[DotNotFailure, R]
-
```

Comment:
does it mean you cannot have type aliases defined in the non-object package?

Reply:
Indeed. 🙂